guides/Identities.md

```diff
@@ -0,0 +1,78 @@
+# About Identities
+
+API calls are restricted using identities. An identity defines who is calling a method and what actions they are authorized to take. 
+Each method can only be called by specific identities. [Notes in the REST reference](#methods-with-restricted-identities) indicate restrictions on the identities that can call a method.  
+Different contexts also limit the identities that can make a given call. For example, apps can only make calls as apps, not as visitors or members.
+
+Wix supports the following identities:
+
+- [Site visitor](#site-visitor) 
+- [Site member](#site-member) 
+- [Wix user](#wix-user) 
+- [Wix app](#wix-app) 
+- [API key admin](#api-key-admin) 
+
+> **Note**: Additional identities are supported for app flows using the [SDK](https://dev.wix.com/docs/sdk).
+
+## Site visitor
+
+An anonymous site visitor who hasn't logged in. Methods that can be called by this identity usually involve operations specific to a particular visitor. These include accessing a list of products, creating and managing a cart, or accessing a login page to authenticate as a member.
+
+**Context**:  
+- [Headless](https://dev.wix.com/docs/go-headless/getting-started/about-headless/about-wix-headless) flows can make calls as site visitors.
```

Comment:
I'm not sure that we want to link the user out to a general Headless article here. We may need to think about how to present the differences in apps/Headless more generally.